app/services/analytics_events.rb

```diff
@@ -5415,6 +5415,10 @@ def second_mfa_reminder_visit
     track_event('Second MFA Reminder Visited')
   end
 
+  def security_check_failed_visited
```

Comment:
Can we add some documentation about what this event is for, so it shows up on our internal handbook Analytics Events page?